src/components/Icons/LinkedIn.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 
 const SvgLinkedIn = props => (
-  <svg width='2em' height='2em' viewBox='0 0 24 24' fill='none' {...props}>
+  <svg width='2em' height='2em' fill='none' {...props}>
     <path
-      d='M18.8136 4H5.18644C4.54237 4 4 4.50848 4 5.15254V18.8475C4 19.4915 4.52542 20 5.18644 20H18.8136C19.4746 20 20 19.4746 20 18.8475V5.15254C20 4.52542 19.4746 4 18.8136 4ZM8.76671 9.97874V17.6119H6.3676V9.97874H8.76671ZM7.56716 6.1791C8.3383 6.1791 8.95522 6.78976 8.95522 7.55308C8.95522 8.3164 8.3383 8.92706 7.56716 8.92706C6.79601 8.92706 6.1791 8.3164 6.1791 7.55308C6.1791 6.78976 6.79601 6.1791 7.56716 6.1791ZM12.4959 10.0074H10.209V17.6716H12.5806V13.8904C12.5806 12.89 12.767 11.9235 14.0036 11.9235C15.2233 11.9235 15.2403 13.0595 15.2403 13.9413V17.6547H17.6119V13.4665C17.6119 11.3978 17.1715 9.8209 14.766 9.8209C13.614 9.8209 12.8517 10.4652 12.5298 11.0587H12.4959V10.0074Z'
+      d='M18.814 4H5.186C4.542 4 4 4.508 4 5.153v13.695C4 19.49 4.525 20 5.186 20h13.628c.66 0 1.186-.525 1.186-1.152V5.152C20 4.525 19.475 4 18.814 4zM8.767 9.979v7.633h-2.4V9.979h2.4zm-1.2-3.8c.771 0 1.388.61 1.388 1.374 0 .763-.617 1.374-1.388 1.374-.771 0-1.388-.61-1.388-1.374 0-.763.617-1.374 1.388-1.374zm4.929 3.828h-2.287v7.665h2.372V13.89c0-1 .186-1.966 1.423-1.966 1.22 0 1.236 1.136 1.236 2.017v3.714h2.372v-4.189c0-2.068-.44-3.645-2.846-3.645-1.152 0-1.914.644-2.236 1.238h-.034v-1.052z'
```

Comment:
Not the same values as the one above?

Reply:
SVGO compresses a little bit (the degrees of precision in the decimals it looks like).  I'm deploying so that we can see if the output is fine.